source/Integrate/Mail_Clients/applemail.md

```diff
@@ -42,5 +42,5 @@ Unfortunately, there is not an easy way to add SMTP API headers to a message in
 {% endinfo %}
 
 {% info %}
-Mail sometimes attaches images in a way that is incompatible with other email readers, but this is [correctable](http://micahgilman.com/play/disable-mac-mailapp-inline-image-attachments/). 
+Mail sometimes attaches images in a way that is incompatible with other email readers, however there are external online resources to help you work around these incompatibilities.
```

Comment:
If we're not linking them to actual resources, we might just want to leave it at: 

Mail can attach images in a way that is incompatible with certain email readers. Test your messages with multiple email clients to avoid this issue.